### ISSUE TITLE
New metadata field: topical tags

### ISSUE THREAD

**Issue by zackbatist** [open-archaeo/publish.open-archaeo.info#2]
As @bbartholdy and others have suggested, it would be very useful to tag each item with terms to denote a journal's focus and scope. There are various ways we might go about this, i.e. whether we should adopt an established list of fields, to what extent we want to standardize and define the topics, etc. Let's discuss these things here.

**Comment by bbartholdy** [comment]
I think we should try to standardise this, otherwise we risk ending up with too many topics that are difficult to filter. Maybe we allow multiple topics to avoid journals that don't fit exactly into a pre-defined list?

**Comment by joeroe** [comment]
A slight downside of using Jekyll is that it only supports a single set of tags and categories, whereas in Hugo for example you can define arbitrary taxonomies and have indexes generated automatically (e.g. one for regional scope, one for period, etc.) With Jekyll we can add extra fields to the front matter as much as we want, as I have for OA model and language and so on, but we'll have to set up any index pages or other filter functionality ourselves.

**Comment by zackbatist** [comment]
I agree that we should allow multiple values to be applied to a record's topical tags.

If we are going to standardize, we also need to be clear about what we mean by "topical tags", since this could include ambiguously-defined regions, periods, fields of study, or other aspects. Although we can separate those out into separate vectors, the table would be littered with null values since journals tend to delimit their scope by only one, maybe two vectors (i.e. either by period, or region, or subfield, but rarely all three at the same time).

Additionally, the specificity of values will differ significantly. For instance, I would define the scope of _Journal of Archaeological Science_ using the broad term "archaeological science", and the journal _Radiocarbon_ with the more specific term "radiocarbon dating", even though "radiocarbon dating" is also "archaeological science". In this example, our terms would still require some intuitive disciplinary knowledge to make sense of, and might even make filtering lead to somewhat misleading or incomplete results.

I imagine this might get quite messy, and could be avoided by using terms that the journals use to describe themselves. This might negate the possibility of implementing search or filter functionality, though. But are these features really necessary for this product?

**Comment by joeroe** [comment]
Maybe it would help to stick to very broad categories, e.g. *JAS* and *Radiocarbon* both go in "archaeological science". The more specific scope of the journal is usually fairly obvious from the title and people can use CTRL+F.

**Comment by bbartholdy** [comment]
How broad would we consider archaeological sciences? Archaeogenetics, zooarchaeology, bioarchaeology, paleobotany?

**Comment by joeroe** [comment]
I would say so, unless we expect to have e.g. more than a couple of diamond journals covering zooarchaeology?

**Comment by bbartholdy** [comment]
Yea good point. That would be great but probably not realistic...
